trustly-android-sdk/build.gradle

```diff
@@ -66,7 +66,7 @@ apply from: 'publish-root.gradle'
 ext {
     PUBLISH_GROUP_ID = 'net.trustly'
     PUBLISH_VERSION = '3.0.0'
-    PUBLISH_ARTIFACT_ID = 'android-sdk'
+    PUBLISH_ARTIFACT_ID = 'trustly-android-sdk'
```

Comment:
Will this be used anywhere visible to a developer user? Just wondering if it should be the same as the repo name or if this is ok. If it's not going to be used as the package name or anything, this is fine 👍

Reply:
We thought that this is an internal reference there in Maven Central, and it should be the same as the package folder or project name